README.md

```diff
@@ -18,31 +17,31 @@ The HERE OLP Edge SDK is a _work in progress_ c++ client for [HERE Open Location
 | :----------------------------- | :------------------------------ |
 | Linux                          | [![Linux code coverage][5]][6]  |
 
-
 [1]: https://travis-ci.com/heremaps/here-olp-edge-sdk-cpp.svg?branch=master
 [2]: https://travis-ci.com/heremaps/here-olp-edge-sdk-cpp
 [3]: https://dev.azure.com/heremaps/github/_apis/build/status/heremaps.here-olp-edge-sdk-cpp?branchName=master
 [4]: https://dev.azure.com/heremaps/github/_build/latest?definitionId=1&branchName=master
 [5]: https://codecov.io/gh/heremaps/here-olp-edge-sdk-cpp/branch/master/graph/badge.svg
 [6]: https://codecov.io/gh/heremaps/here-olp-edge-sdk-cpp/
 
-## Why Use the SDK
+## Why Use
 
-The HERE OLP Edge SDK C++ provides support for core **HERE Open Location Platform (OLP)** use cases through a set of native C++ interfaces. The SDK should save its users time and effort using OLP REST APIs and provides a set of stable APIs simplifying complex OLP operations and removing the burden of keeping up to date with the latest OLP REST API changes.
-It is a modern (C++11 or later), lightweight, and modular SDK with minimal dependencies targeted towards a wide range of hardware platforms from embedded devices to desktops.
+The HERE OLP Edge SDK C++ provides support for core HERE Open Location Platform (OLP) use cases through a set of native C++ interfaces. The SDK is intended to save your time and effort on using OLP REST APIs and provides a set of stable APIs that simplify complex OLP operations and remove the burden of keeping up to date with the latest OLP REST API changes.
```

Comment:
```suggestion
The HERE OLP Edge SDK C++ provides support for core HERE Open Location Platform (OLP) use cases through a set of native C++ interfaces. The SDK is intended to save your time and effort on using OLP REST APIs and provides a set of stable APIs that simplify complex OLP operations and removes the burden of keeping up to date with the latest OLP REST API changes.
```

Reply:
I used "remove" to relate to "stable APIs" as continuation to "that simplify... and". This way, the focus is on _stability_ of APIs, not on the fact that it's a _set_ of APIs, which I believe is the case. @leonid-ziskel - Please verify if my point is correct.